active_directory/pyproject.toml

```diff
@@ -1,7 +1,8 @@
 [build-system]
 requires = [
     "hatchling>=0.11.2",
-    "setuptools<61",
+    "setuptools>=66; python_version > '3.0'",
+    "setuptools; python_version < '3.0'",
```

Comment:
Should this have an upper limit on versioning for Python < 3.0? Looks like In `setuptools=47.0.0` changelog, it mentioned that Python 2 users should use `setuptools<45`

Reply:
it won't pick up a version that is not available for py2